test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
ignore folder 'test' in file '.gitignore'. You don't need follow 'test' in turtorial anymore